### ISSUE TITLE
Map plots does not load tiles correctly

### ISSUE THREAD

**Issue by kobbas** [wolkesson/SampleCrunch#39]
The map plot does not plot any map using lo/la degree coordinates. Thus, with previous working files it is not possible to plot a trace on map any more. I guess that OSM have done some changes in their API.

Regrards,
Edvin

**Comment by wolkesson** [comment]
Yes, OSM now require a User Agent to be provided with each request and maximum 2 download threads. Please active Pre-releases and upgrade to 1.0.99.00. If this works for you too, I'll make a new release.
